Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1665,3 +1671,31 @@ + (NSString *)kitDisplayVersion {
 }
 
 @end
+
+#pragma mark - BranchNetworkServiceClass
+
+static Class bnc_networkServiceClass = NULL;
+
+void  BranchSetNetworkServiceClass(Class networkServiceClass) {
```

Comment:
This pattern of setter doesn't conform with the rest of our setter APIs. May I recommend `setCustomNetworkServiceClass`?